pvlib/iotools/tmy.py

```diff
@@ -198,9 +217,17 @@ def read_tmy3(filename, coerce_year=None, recolumn=True):
     # NOTE: as of pvlib-0.6.3, min req is pandas-0.18.1, so pd.to_timedelta
     # unit must be in (D,h,m,s,ms,us,ns), but pandas>=0.24 allows unit='hour'
     data.index = data_ymd + pd.to_timedelta(shifted_hour, unit='h')
-
-    if recolumn:
-        data = _recolumn(data)  # rename to standard column names
+    if map_variables is None:
+        if recolumn:
+            data = _recolumn(data)  # rename to standard column names
```

Comment:
I don't think this is what we want.  If the user specifies `map_variables=False` as suggested, there is no way to activate the `recolumn` functionality.

Reply:
Fair. I'd be ok with changing the first line to:

    if map_variables is not True:

Reply:
@AdamRJensen then should the default be changed to False ?

Reply:
No, I still think leaving the default value to None is a good intermediary step as we eventually will change it to True in 0.11.

Reply:
I think my comment here is still unaddressed.